apps/emqx/src/emqx_modules.erl

```diff
@@ -80,12 +80,14 @@ unload(ModuleName) ->
 reload(emqx_mod_acl_internal) ->
     Modules = emqx:get_env(modules, []),
     Env = proplists:get_value(emqx_mod_acl_internal, Modules, undefined),
-    case emqx_mod_acl_internal:reload(Env) of
+    Reload = emqx_mod_acl_internal:reload(Env),
+    case Reload of
         ok ->
             ?LOG(info, "Reload ~s module successfully.", [emqx_mod_acl_internal]);
         {error, Error} ->
             ?LOG(error, "Reload module ~s failed, cannot start for ~0p", [emqx_mod_acl_internal, Error])
```

Comment:
I believe it is intended for here to return ok.
the only error reason is `already_exists` which is harmless.
changing it to error may cause the caller to crash.